package-lock.json

```diff
@@ -2415,6 +2415,7 @@
 			"version": "file:packages/rich-text",
 			"requires": {
 				"@babel/runtime": "^7.0.0",
+				"@wordpress/escape-html": "file:packages/escape-html",
```

Comment:
This was probably just something that someone forgot to merge, I saw it in another PR I reviewed recently too (
https://github.com/WordPress/gutenberg/pull/10295/files#r222124781)

Reply:
Yes, I’m that person :)